vaadin-infinite-scroller.html

```diff
@@ -156,7 +156,8 @@
 
     ready: function() {
       this._bufferHeight = this.itemHeight * this.bufferSize;
-      this._buffers = Polymer.dom(this.root).querySelectorAll('.buffer');
+      let nodes = Polymer.dom(this.root).querySelectorAll('.buffer');
+      this._buffers = [].slice.call(nodes);
```

Comment:
Fixes a Polymer 2 bug where an array methods are called on `nodes` which is a `nodeList` that is Array-like.